crates/control/src/ball_filter.rs

```diff
@@ -45,7 +45,6 @@ pub struct CycleContext {
     pub current_odometry_to_last_odometry:
         HistoricInput<Option<Isometry2<f32>>, "current_odometry_to_last_odometry?">,
     pub historic_camera_matrices: HistoricInput<Option<CameraMatrices>, "camera_matrices?">,
-    pub projected_limbs: HistoricInput<Option<ProjectedLimbs>, "projected_limbs?">,
```

Comment:
What did you do here? It is probably the easiest if you shortly explain instead of me digging deep into the details and trying to understand.

Reply:
The limb projector was moved from Control to Vision. I think both are fine and I have no strong opinion regarding this.